src/ExperimentalInternetComputer.mo

```diff
@@ -13,6 +13,17 @@ module {
   /// * The error code and textual message data of an IC reject determines the future's `Error` value.
   ///
   /// Note: `call` is an asynchronous function and can only be applied in an asynchronous context.
+  ///
+  /// Example:
+  /// ```motoko
+  /// import IC "mo:base/ExperimentalInternetComputer";
+  /// import Principal "mo:base/Principal";
+  ///
+  /// let ledger = Principal.fromText("ryjl3-tyaaa-aaaaa-aaaba-cai");
+  /// let method = "decimals";
+  /// let input = ();
+  /// let rawReply = await IC.call(ledger, method, to_candid(input));
```

Comment:
Would it make sense to give an example for decoding the raw reply, or is this subject to change?

Reply:
We can add `from_candid(rawReply)` in the example.

Reply:
I built this example in Motoko Playground, and I couldn't seem to find the type annotation expected for decoding `rawReply`. Is this documented somewhere?

Reply:
https://internetcomputer.org/docs/current/developer-docs/build/cdks/motoko-dfinity/language-manual#candid-serialization?
`rawReply` is a blob, so the expected type will be the return type of `ic.decimals()`. Something like `let ret : ?{ decimals:Nat32 } = from_candid(rawReply);`